Formula/kubernetes-cli.rb

```diff
@@ -13,7 +13,7 @@ class KubernetesCli < Formula
     sha256 "a97a579a2adaa9720aa122355b0a43cbebea49da394f3489a300f41d8d5e7f77" => :high_sierra
   end
 
-  depends_on "go@1.12" => :build
```

Comment:
Latest k8s is using go 1.13, relates to https://github.com/kubernetes/kubernetes/pull/82809